config.py

```diff
@@ -31,6 +32,9 @@ class Default(object):
     # Configure JWT error message key
     JWT_ERROR_MESSAGE_KEY = 'message'
 
+    # Configure CORS origins
+    CORS_ORIGINS = re.split(r'[,;\s]', os.environ.get('CORS_ORIGINS'))
```

Comment:
When I run the app without the env var set I get a cryptic error message returned. 
```
TypeError: expected string or bytes-like object
```
This is because the get method returns None and passes it into the regex. 
If we change this:
`os.environ.get('CORS_ORIGINS'))`
to this: 
`os.environ['CORS_ORIGINS'])`
Then the user will be informed that they're missing a the key CORS_ORIGINS
```
KeyError: 'CORS_ORIGINS'
```
Still not that informative, but a bit better I think

Reply:
The regex seems a bit complex. I think having a test here would be valuable and would accomplish a couple things:
1. It would verify this works as intended
2. It would act as documentation to show what values could be inputted and the expected outputted values

I think Number 2 is important, because there is nothing to show whoever deploys the app what value to put in for CORS_ORIGINS. They might simply do '*' as the `env.example` shows.

Reply:
CORS origins are now configurations and not in the environmental variable. So this allows us to get rid of the reg ex.